composer.json

```diff
@@ -19,7 +19,8 @@
     },
     "require-dev": {
         "phpunit/phpunit": "^4.5 || ^5.4",
-        "guzzlehttp/psr7": "^1.3"
+        "guzzlehttp/psr7": "^1.3",
+        "php-http/message": "^1.0"
```

Comment:
so if i install the stream wrapper with client-common 1.0 (and not message explicitly) will things work or not?